app/controllers/documents_controller.rb

```diff
@@ -9,6 +9,7 @@ class DocumentsController < AuthenticationController
   before_action :ensure_document_edits_unlocked, only: %i[new edit update archive]
 
   def index
+    @invalid_documents = @planning_application.documents.where(validated: false).where(archived_at: nil)
```

Comment:
Possibly move this to a scope in the model?

Reply:
I have always seen .where clauses used in controller methods so thought it was best practice - we have another one for ex in published? on line 57. What would be the logic for placing it in the model ( also considering this one is only needed for the index page) ?